src/app/static/src/tests/components/runReport/changeLog.test.ts

```diff
@@ -0,0 +1,80 @@
+import {mount} from "@vue/test-utils";
+import ChangeLog from "../../../js/components/runReport/changeLog.vue";
+import changeLog from "../../../js/components/runReport/changeLog.vue";
+
+describe(`changeLog`, () => {
+    const changelogTypeOptions = ["internal", "public"]
+
+    const getWrapper = () => {
+        return mount(ChangeLog,
+            {
+                propsData: {
+                    showChangelog: true,
+                    changelogTypeOptions: changelogTypeOptions
+                }
+            })
+    }
+
+    it(`renders component as expected`, () => {
```

Comment:
Could also check that it emits its initial type selection event, if it has any types.